package.json

```diff
@@ -23,11 +23,11 @@
     "proseWrap": "always"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free": "6.3.0",
+    "@fortawesome/fontawesome-free": "6.4.0",
     "bootstrap": "5.2.3"
   },
   "devDependencies": {
-    "hugo-extended": "0.110.0"
+    "hugo-extended": "0.112.3"
```

Comment:
Darn, I missed this update to `hugo-extended`, and it's causing build issues.

In general, @deining please avoid slipping in an unrelated change (the title of this PR is about FA), or make it very clear in the tilte and/or description that there's more.

I'm going to revert this change and address the Hugo update separately.